elements/urth-core-behaviors/dynamic-properties-behavior.html

```diff
@@ -28,6 +32,12 @@
             addDynamicProperties: function(dynProps){
                 this._dynamicProperties = dynProps;
```

Comment:
Unrelated to this change, but seems like this method should really be `setDynamicProperties` since `this._dynamicProperties` is reset on each invocation instead of actually "adding".